azure/templates/pds-tests.yml

```diff
@@ -48,3 +48,30 @@ steps:
         $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/tests/test-report.xml
         $(Pipeline.Workspace)/s/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/tests/*.xml
       failTaskOnFailedTests: true
+
+  - bash: |
+      pip install virtualenv
```

Comment:
Poetry is already being used to manage envs and packages (which should already be installed), so this step isn't required